backend/src/v5/models/modelSettings.js

```diff
@@ -62,14 +98,19 @@ Models.getFederationById = async (ts, federation, projection) => {
 	}
 };
 
-Models.getContainers = async (ts, ids, projection, sort) => {
+Models.getModelByName = async (ts, ids, name, projection) => {
```

Comment:
This function title doesn't really match the name. It's more like get model by name within this limited set of ids, and it only returns you one of them.

It might be better to just expose a findOneByQuery function, as this is pretty bespoke.